test/functional/plugin/lsp_spec.lua

```diff
@@ -861,4 +861,29 @@ describe('LSP', function()
       ]])
     end)
   end)
+  describe('lsp.util.show_line_diagnostics', function()
+    it('creates floating window and returns popup bufnr and winnr if current line contains diagnostics', function()
+      eq(3, exec_lua [[
+        BUFFER = vim.api.nvim_create_buf(false, true)
```

Comment:
generally better to use `local` unless there is some technical reason a global is needed.
```suggestion
        local BUFFER = vim.api.nvim_create_buf(false, true)
```